sled-hardware/src/underlay.rs

```diff
@@ -70,3 +70,37 @@ pub fn find_chelsio_links() -> Result<Vec<PhysicalLink>, Error> {
             .collect())
     }
 }
+
+/// Return links to delegate to the switch zone for transmit mode maghemite
+pub fn find_switch_zone_nics() -> Result<Vec<AddrObject>, Error> {
```

Comment:
If I'm reading `find_switch_zone_links_for_maghemite` correctly, this will always return an empty `Vec`, unless the code is changed?

The comment about "no delegation required" below doesn't really make this clear IMO

Reply:
See 80afc01e